stories/2-elements/buttons.stories.js

```diff
@@ -5,6 +5,25 @@ import omit from 'storybook-react-omit'
 import Button from '../../packages/components/lib/Button'
 import bp from '../../packages/components/lib/breakpoints'
 
+const primaryInfoText = `
+  # BUTTONS
+  Buttons are a very common component used throughout CG interfaces. Almost all calls-to-action are accompanied by one or more buttons. Because of their frequent use, there are several styles to choose from and they should be selected with functionality in mind.
+
```

Comment:
This Should be H2 rather than H1. Since, all other places it has started with H2.